framework/logging.cpp

```diff
@@ -1221,6 +1246,33 @@ void log_platform_message(const char *fmt, ...)
     log_message_preformatted(-1, msg);
 }
 
+#undef log_message_skip
+void log_message_skip(int thread_num, SkipCategory category, const char *fmt, ...)
+{
+    if (current_output_format() == SandstoneApplication::OutputFormat::no_output)
+        return;
+
+    assert_log_message(fmt);
+    va_list va;
+    va_start(va, fmt);
+    std::string msg = create_filtered_message_string(fmt, va);
+    va_end(va);
+
+    std::string tmp_s;
+    tmp_s += SANDSTONE_LOG_SKIP;
```

Comment:
Please remove the call to `assert_log_message` and starting with `SANDSTONE_LOG_SKIP`.

Reply:
Done.